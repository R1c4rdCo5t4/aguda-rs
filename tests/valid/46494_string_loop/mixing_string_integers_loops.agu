-- 46494, Leonor Figueirinhas

let _ : Unit = 
    let message : String = "H3ll0 W0rld";
    print(message);

    let str : String[] = new String[] [11 | ""];
        set str[0] = "H";
        set str[1] = "3";
        set str[2] = "L";
        set str[3] = "L";
        set str[4] = "0";
        set str[5] = " ";
        set str[6] = "W";
        set str[7] = "0";
        set str[8] = "R";
        set str[9] = "L";
        set str[10] = "D";

        print(length(str));
        let i : Int = 0;
        while i < length(str) do (
            print (str[i]);
            set i = i + 1
        )
