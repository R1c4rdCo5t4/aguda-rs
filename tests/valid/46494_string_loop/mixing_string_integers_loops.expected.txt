H3ll0 W0rld
11
H
3
L
L
0

W
0
R
L
D